stable_baselines3/common/preprocessing.py

```diff
@@ -56,10 +53,10 @@ def is_image_space(
         if not check_channels:
             return True
         # Check the number of channels
-        if channels_last:
-            n_channels = observation_space.shape[-1]
-        else:
+        if is_image_space_channels_first(observation_space):
```

Comment:
Just a thought: I was about to comment that maybe we should not rely on this, but then realized that if somebody has height/width < channels, it probably breaks in CNNs and whatnot. All good 👍